### ISSUE TITLE
Raspberry Twitch API where is the Config File?

### ISSUE THREAD

**Issue by PasiiP** [streamlink/streamlink#2763]
Where did I find the livestreamer config file on a raspberry pi??

**Comment by bastimeyer** [comment]
This is Streamlink, not the Livestreamer repository.
Not sure what you mean with Twitch API, but the documentation regarding Streamlink's config file can be found here:
https://streamlink.github.io/cli.html#configuration-file

**Comment by PasiiP** [comment]
Hab eben gesehen das du Deutsch bist, ich hoffe es ist okay wenn ich deutsch schreibe. Ich habe streamlink auf meinem Raspberry Pi installiert versuche es auszuführen bekomme dann aber ein Fehler wegen der Twitch API und soll diese in der Config von Streamlink  hinzufügen allerdings finde ich diese weder im Ordner von Streamlink sonst noch irgendwo?

**Comment by bastimeyer** [comment]
The language of this issue tracker is English. I don't want to make any exceptions here, sorry.

To answer your question, if you're getting errors while trying to watch Twitch.tv streams, then you're simply using an outdated version of Streamlink. There are no workarounds required if you're using the latest version, but since you didn't include a debug log in your issue, I don't know which version you're currently using.
Also, if you're trying to watch Twitch.tv clips, then you're out of luck, since this hasn't been fixed yet (#2752).

I don't own a RaspPi myself, but it's possible that the repositories are outdated or that you didn't update yet. If there's no Streamlink 1.3.0 available, then you can install Streamlink from `pip`. See the install documentation on Streamlink's website.
https://streamlink.github.io/install.html#source-code

And regarding the config file, as the documentation already points out on the link I've posted earlier, Streamlink doesn't come with a pre-defined config file on *nix systems. This is specific to the Windows installer.

The config file, if it exists, gets read from one of these paths in this order, according to the XDG base directory specification:
```bash
${XDG_CONFIG_HOME}/streamlink/config
${HOME}/.config/streamlink/config
${HOME}/.streamlinkrc
```
